mern/client/src/Components/Navbar/navState.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+export const NavStateContext = React.createContext()
+
+export default function NavState({ children }) {
+    // const NavStateContext = React.createContext()
+    const [navState, setNavState] = React.useState(false)
+
+
+    return (
+        <NavStateContext.Provider value={[navState, setNavState]}>
```

Comment:
Perhaps change the choice of NavState to something a bit more fitting. Maybe like SessionContext because it could potentially store more information about the session in the future)

Reply:
Good point. Would like to add the Session ID to the context later on. Will go with SessionContext